README.md

```diff
@@ -2,34 +2,140 @@
 
 ## Getting started
 
-Fork and clone
+Fork and then clone
+
+Navigate into project
 
 ```shell
 cd bobabot
 ```
+Initialize your python virtual environment
 
-Navigate into project
+_NOTE_: This is only necessary for new environment setups_
 
 ```shell
 python -m venv venv
 ```
 
-Initialize your python virtual environment
-
-_note: This is only necessary for new environment setups_
+Activate the python virtual environment
 
 ```shell
 source venv/bin/activate
 ```
 
-Activate the python virtual environment
+Install project dependencies into virtual environment
 
 ```shell
 pip install -r requirements.txt
 pip install -r requirements.dev.txt
 ```
 
-Install project dependencies into virtual environment
+### Creating a discord bot & server
+
+First, create a new server
+
+Enter server settings
+
+Under Community, register your server as a community server
+
+Add a new forum text channel, make the name match your .env variable *DISCORD_CHANNEL_NAME*
+
+Under the settings for this new channel, add a couple tags. These represent different fields/roles.
+
+```shell
+# Example Tags
+SWE, PM, DS, UI/UX
+```
+
+Additionally, register for a discord developer account and create a new bot application.
+
+Invite your bot to your server with the following set of permissions:
+
+- Manage Server
+- Manage Channels
+- Send Messages
+- Create Public Threads
+- Manage Messages
+- Embed Links
+- Use Slash Commands
+
+_NOTE_: Generate a new token under the *bot* tab and keep it, it will be used later in the .env file
+
+
+### Docker & MongoDB
+
+Ensure [Docker](https://docs.docker.com/get-docker/) is installed
+
+Open Terminal/Command Prompt
+
+Pull MongoDB
+
+```shell
+docker pull mongo:latest
+```
+
+Start a mongoDB server bound to port 27017 (Or a port of your choosing, provided you update your localhost address in .env later).
+
+```shell
+docker run -d -p 27017:27017 --name bobabot mongo:latest
+```
+
+To ensure that our port and container are running properly, in the command prompt write:
+
+```shell
+docker ps
+```
+
+This should return a list of active containers, and under port you will find listed 0.0.0.0:27017 (or your custom port address)
+
+Next, we need to configure our .env file to ensure we connect to the correct container when running our local development server.
+
+Within your repository, create a file named ".env" with the following
+
+```shell
+# Port should match your container
+MONGO_URI='mongodb://localhost:27017/'
+# Database name is your container name
+DATABASE_NAME='bobabot'
+# To create a token, register on discord for developers. Create a new application, and see under the bot tab. Copy paste your token here.
+DISCORD_TOKEN='abCd214sampleToken'
+# Set to the name of your testing discord
+DISCORD_SERVER_NAME='sample_server_name'
+# Set to the name of your testing community "forum" channel
+DISCORD_CHANNEL_NAME='resumes'
+SERVER_URL='http://localhost:5000'
```

Comment:
Can we commit this to a .env.sample or sample.env? Then we can leave this out of here, env files are pretty common now and we can reduce the instructions to copying the sample file to `.env` and add in any secrets there

Reply:
I can do that, but I don't think it would change much. I think generally this wouldn't change the onboarding or testing flow at all, the person still has to copy it from one location to another